UserService/app/src/main/java/com/bookmyshow/userservice/dao/entity/Permission.java

```diff
@@ -0,0 +1,60 @@
+package com.bookmyshow.userservice.dao.entity;
+
+import javax.persistence.*;
+import java.util.List;
+import java.util.Set;
+
+@Table(name = "PERMISSION")
+@Entity
+public class Permission {
```

Comment:
Do we need to implement Serializable? Why do we need to implement Serializable, does it require here? Other two entities are implemented and not to this. Pls explore...Serialization, Externalization concepts, and need of it in Hibernate Entities.